magic-links/.env

```diff
@@ -0,0 +1,23 @@
+# required: true
+# format: sid
+# description: Your Twilio Account SID. Can be found in the Console.
+# link: https://www.twilio.com/console
+TWILIO_ACCOUNT_SID=
+
+# required: true
+# format: secret
+# description: Your Twilio Auth Token. Can be found in the Console.
+# link: https://www.twilio.com/console
+TWILIO_AUTH_TOKEN=
+
+# description: SID of your Twilio Verify Service
+# format: sid
+# link: https://www.twilio.com/console/verify/services
+# required: true
+VERIFY_SERVICE_SID=
+
+# description: URL used by the email template to check verifications
+# format: url
+# required: true
+# default: http://localhost:3000/verify.html
```

Comment:
While this is valid according to the new schema, the preferred way would be to add it in the line below such as: `CALLBACK_URL=http://localhost:3000/verify.html`

I'd also consider to avoid full hard-coded URLs and instead consider using `context.DOMAIN_NAME` and `context.PATH` as much as possible to construct URLs because any of these templates could be added to an existing project as a nested template using `twilio serverless:new some-namespace --template magic-link`